sympy/stats/joint_rv_types.py

```diff
@@ -242,12 +259,26 @@ def NormalGamma(syms, mu, lamda, alpha, beta):
     Parameters
     ==========
 
-    syms: list/tuple/set of two symbols for identifying each component
+    sym: A symbol/str for identifying the random variable. This can be
```

Comment:
Is the second sentence here correct? I'm assuming the original "syms" was wrong (also the parameter is still called `syms`, not `sym`.

Reply:
Thanks, I'll change `sym` to `syms`. The second sentence meaning the one talking about indexing? I think that's correct as it can be used to calculate the marginal density using `marginal_density` function. Do you think it should be changed?

Reply:
"component of the joint probability space" can be confusing. I'll remove that part.